src/lib/cpu_info/cpu_info_funcs.c

```diff
@@ -0,0 +1,11 @@
+#include <lib/libcpu_info.h>
+#include <string.h>
+#include <assert.h>
+
+extern void set_feature_val(struct cpu_info *info, const char *name, unsigned int val) {
+    int current_feature = info->feature_count;
+	assert((current_feature < MAX_NUM_FEATURES) && "Exceeded Number of Features!");
```

Comment:
I see mixing of tabs and spaces here, use only tabs, please.

Reply:
Really Sorry! I copy-pasted and this happened probably due to my text editor's settings. Will fix.